packages/components/src/public/files/FileAttachmentForm.tsx

```diff
@@ -408,7 +411,10 @@ export class FileAttachmentForm extends PureComponent<FileAttachmentFormProps, S
                         <div className="col-md-3">
                             <div className="pull-right">
                                 {renderTemplateButton && (
-                                    <TemplateDownloadButton templateUrl={this.props.templateUrl} />
+                                    <TemplateDownloadButton
+                                        defaultTemplateUrl={this.props.templateUrl}
+                                        schemaQuery={this.props.templateSchemaQuery}
```

Comment:
should the `templateSchemaQuery` prop be added to the `shouldRenderTemplateButton` check?

Reply:
I don't think it's needed. If defaultTemplateUrl is null, we don't want to show download button at all. This is the case for certain registries.